app/controllers/staff/activities/uploads_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 require "csv"
 
-class Staff::ActivityUploadsController < Staff::BaseController
+class Staff::Activities::UploadsController < Staff::BaseController
```

Comment:
Are we level namespacing these or are we only level namespacing level B, with levels C and D as defaults?

Reply:
Makes sense - this question applies to namespacing across the board, but I think we can probably treat most/all of it the same way

Reply:
Yeah `sub_level_b` is a bit... something